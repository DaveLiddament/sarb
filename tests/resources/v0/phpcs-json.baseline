{"historyAnalyser":"git","historyMarker":"0f078b7d445f661991605b91d8ddb9c7f796242d","resultsParser":"phpcodesniffer-json","analysisResults":[{"lineNumber":2,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Squiz.Commenting.FileComment.Missing","message":"Missing file doc comment","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"message\":{\"message\":\"Missing file doc comment\",\"source\":\"Squiz.Commenting.FileComment.Missing\",\"severity\":5,\"fixable\":false,\"type\":\"ERROR\",\"line\":2,\"column\":1}}"},{"lineNumber":7,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Squiz.Commenting.ClassComment.Missing","message":"Missing doc comment for class InvalidPathException","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"message\":{\"message\":\"Missing doc comment for class InvalidPathException\",\"source\":\"Squiz.Commenting.ClassComment.Missing\",\"severity\":5,\"fixable\":false,\"type\":\"ERROR\",\"line\":7,\"column\":1}}"},{"lineNumber":9,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Squiz.Commenting.FunctionComment.Missing","message":"Missing doc comment for function __construct()","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"message\":{\"message\":\"Missing doc comment for function __construct()\",\"source\":\"Squiz.Commenting.FunctionComment.Missing\",\"severity\":5,\"fixable\":false,\"type\":\"ERROR\",\"line\":9,\"column\":1}}"},{"lineNumber":11,"fileName":"src\/Domain\/Common\/InvalidPathException.php","type":"Generic.Files.LineLength.TooLong","message":"Line exceeds 85 characters; contains 109 characters","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/src\\\/Domain\\\/Common\\\/InvalidPathException.php\",\"message\":{\"message\":\"Line exceeds 85 characters; contains 109 characters\",\"source\":\"Generic.Files.LineLength.TooLong\",\"severity\":5,\"fixable\":false,\"type\":\"WARNING\",\"line\":11,\"column\":1}}"},{"lineNumber":8,"fileName":"src\/Domain\/BaseLiner\/BaseLineImporter.php","type":"Generic.Files.LineLength.TooLong","message":"Line exceeds 85 characters; contains 113 characters","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/src\\\/Domain\\\/BaseLiner\\\/BaseLineImporter.php\",\"message\":{\"message\":\"Line exceeds 85 characters; contains 113 characters\",\"source\":\"Generic.Files.LineLength.TooLong\",\"severity\":5,\"fixable\":false,\"type\":\"WARNING\",\"line\":8,\"column\":1}}"},{"lineNumber":52,"fileName":"src\/Domain\/BaseLiner\/BaseLineImporter.php","type":"Squiz.WhiteSpace.FunctionSpacing.Before","message":"Expected 18 spaces after parameter type; 1 found","fullDetails":"{\"absoluteFilePath\":\"\\\/vagrant\\\/src\\\/Domain\\\/BaseLiner\\\/BaseLineImporter.php\",\"message\":{\"message\":\"Expected 18 spaces after parameter type; 1 found\",\"source\":\"Squiz.WhiteSpace.FunctionSpacing.Before\",\"severity\":5,\"fixable\":true,\"type\":\"ERROR\",\"line\":52,\"column\":1}}"}]}