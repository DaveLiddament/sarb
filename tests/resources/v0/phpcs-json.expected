{
    "historyAnalyser": "git",
    "historyMarker": "0f078b7d445f661991605b91d8ddb9c7f796242d",
    "resultsParser": "phpcodesniffer-json",
    "analysisResults": [
        {
            "lineNumber": 2,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Squiz.Commenting.FileComment.Missing",
            "message": "Missing file doc comment"
        },
        {
            "lineNumber": 7,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Squiz.Commenting.ClassComment.Missing",
            "message": "Missing doc comment for class InvalidPathException"
        },
        {
            "lineNumber": 9,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Squiz.Commenting.FunctionComment.Missing",
            "message": "Missing doc comment for function __construct()"
        },
        {
            "lineNumber": 11,
            "fileName": "src\/Domain\/Common\/InvalidPathException.php",
            "type": "Generic.Files.LineLength.TooLong",
            "message": "Line exceeds 85 characters; contains 109 characters"
        },
        {
            "lineNumber": 8,
            "fileName": "src\/Domain\/BaseLiner\/BaseLineImporter.php",
            "type": "Generic.Files.LineLength.TooLong",
            "message": "Line exceeds 85 characters; contains 113 characters"
        },
        {
            "lineNumber": 52,
            "fileName": "src\/Domain\/BaseLiner\/BaseLineImporter.php",
            "type": "Squiz.WhiteSpace.FunctionSpacing.Before",
            "message": "Expected 18 spaces after parameter type; 1 found"
        }
    ]
}